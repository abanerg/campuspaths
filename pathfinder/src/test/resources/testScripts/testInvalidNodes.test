# Create graph

CreateGraph graph

AddNode graph A
AddNode graph B
AddNode graph C
AddNode graph D

# Should give unknown: because nodes asked for are not on graph

FindPath graph X Y
