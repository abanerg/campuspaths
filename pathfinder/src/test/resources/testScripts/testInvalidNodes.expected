# Create graph

created graph graph

added node A to graph
added node B to graph
added node C to graph
added node D to graph

# Should give unknown: because nodes asked for are not on graph

unknown: X
unknown: Y
